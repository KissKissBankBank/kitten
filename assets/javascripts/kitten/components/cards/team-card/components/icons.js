import React from 'react'
import styled from 'styled-components'
import PropTypes from 'prop-types'
import { pxToRem } from '../../../../helpers/utils/typography'
import { ButtonIcon } from '../../../buttons/button-icon'
import { LinkedinIcon } from '../../../icons/linkedin-icon'
import { TwitterIcon } from '../../../icons/twitter-icon'
import { EmailIcon } from '../../../icons/email-icon'
import { TeamCardPhoneIcon } from './phone-icon'
import deprecated from 'prop-types-extra/lib/deprecated'

const StyledTeamCardIcons = styled.div`
  margin-top: ${pxToRem(15)};
  display: flex;

  > * {
    flex-shrink: 0;
  }

  .k-TeamCardIcons__buttonIcon:not(:last-child) {
    margin-right: ${pxToRem(15)};
  }
`

export const TeamCardIcons = ({
  email,
  phoneNumber,
  links,
  socialLink,
  ...props
}) => {
  const getSocialLinks = () => {
    if (links.length > 0) return links

    // handle deprecated `socialLink` prop
    if (socialLink) return [{ name: 'linkedin', href: socialLink }]
  }

  return (
    <StyledTeamCardIcons>
      {email && (
        <ButtonIcon
          size="tiny"
          tag="a"
          href={`mailto:${email}`}
          modifier="hydrogen"
          className="k-ButtonIcon--phone k-TeamCardIcons__buttonIcon"
          aria-label="E-mail"
        >
          <EmailIcon aria-hidden className="k-ButtonIcon__svg" />
        </ButtonIcon>
      )}

      {phoneNumber && (
        <TeamCardPhoneIcon
          {...props}
          phoneNumber={phoneNumber}
          className="k-TeamCardIcons__buttonIcon"
        />
      )}

      <SocialLinks links={getSocialLinks()} />
    </StyledTeamCardIcons>
  )
}

const SocialLinks = ({ links = [] }) => (
  <>
    {links.map(link => {
      if (!link.href) return

      return (
        <ButtonIcon
          tag="a"
          key={link.name}
          aria-label={link.name}
          href={link.href}
          modifier={`social_${link.name}`}
          target="_blank"
          rel="noopener"
          className="k-TeamCardIcons__buttonIcon"
          size="tiny"
        >
          {link.name === 'linkedin' && (
            <LinkedinIcon aria-hidden className="k-ButtonIcon__svg" />
          )}
          {link.name === 'twitter' && (
            <TwitterIcon aria-hidden className="k-ButtonIcon__svg" />
          )}
        </ButtonIcon>
      )
    })}
  </>
)

TeamCardIcons.propTypes = {
  email: PropTypes.string,
  phoneNumber: PropTypes.string,
  links: PropTypes.arrayOf(
    PropTypes.shape({
      name: PropTypes.oneOf(['linkedin', 'twitter']),
      href: PropTypes.string,
    }),
  ),

  // Deprecated.
  socialLink: deprecated(PropTypes.string, 'Prefer use links prop'),
}

TeamCardIcons.defaultProps = {
  email: '',
  phoneNumber: '',
  links: [],
}
