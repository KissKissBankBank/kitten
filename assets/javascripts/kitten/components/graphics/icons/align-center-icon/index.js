import React from 'react'
import PropTypes from 'prop-types'

export const AlignCenterIcon = ({ color, title, ...props }) => (
  <svg
    width="14"
    height="10"
    viewBox="0 0 14 10"
    xmlns="http://www.w3.org/2000/svg"
    {...props}
  >
    {title && <title>{title}</title>}
    <path d="M12 8v2H3V8h9zm2-4v2H0V4h14zm-1-4v2H2V0h11z" fill={color} />
  </svg>
)

AlignCenterIcon.propTypes = {
  color: PropTypes.string,
  title: PropTypes.string,
}

AlignCenterIcon.defaultProps = {
  color: '#222',
  title: '',
}
