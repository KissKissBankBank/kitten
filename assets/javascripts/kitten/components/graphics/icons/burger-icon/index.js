import React from 'react'
import classNames from 'classnames'
import PropTypes from 'prop-types'
import styled, { css } from 'styled-components'
import COLORS from '../../../../constants/colors-config'
import deprecated from 'prop-types-extra/lib/deprecated'

const StyledItem = styled.svg`
  overflow: visible;

  fill: var(--burgerIcon-mainColor, ${COLORS.font1});

  :hover,
  button:hover & {
    fill: var(--burgerIcon-hoverColor, ${COLORS.primary1});
  }

  rect {
    transition: transform 0.2s ease-out, fill 0.15s;
  }

  &.k-BurgerIcon--isActive {
    .k-BurgerIcon__bun {
      transform: translateX(2px);
    }
    .k-BurgerIcon__patty {
      transform: translateX(-2px);
    }
  }

  &.k-BurgerIcon--isAnimatedOnHover {
    &:hover,
    button:hover &,
    &:focus,
    button:focus & {
      .k-BurgerIcon__bun {
        transform: translateX(2px);
      }
      .k-BurgerIcon__patty {
        transform: translateX(-2px);
      }
    }
  }
`

export const BurgerIcon = ({
  mainColor,
  hoverColor,
  isActive,
  iconTitle,
  title,
  width,
  height,
  isAnimatedOnHover,
  style,
  className,
  ...props
}) => (
  <StyledItem
    role="img"
    aria-label={iconTitle || title}
    xmlns="http://www.w3.org/2000/svg"
    viewBox="0 0 12 10"
    width={width}
    height={height}
    style={{
      ...style,
      '--burgerIcon-mainColor': mainColor,
      '--burgerIcon-hoverColor': hoverColor,
    }}
    className={classNames(
      'k-BurgerIcon',
      className,
      {
        'k-BurgerIcon--isActive': isActive,
        'k-BurgerIcon--isAnimatedOnHover': isAnimatedOnHover,
      }
    )}
    {...props}
  >
    {(iconTitle || title) && <title>{iconTitle || title}</title>}
    <rect y="0" width="12" height="2" className="k-BurgerIcon__bun" />
    <rect y="4" width="12" height="2" className="k-BurgerIcon__patty" />
    <rect y="8" width="12" height="2" className="k-BurgerIcon__bun" />
  </StyledItem>
)

BurgerIcon.propTypes = {
  mainColor: PropTypes.string,
  hoverColor: PropTypes.string,
  isActive: PropTypes.bool,
  iconTitle: deprecated(PropTypes.string, 'Prefere use `title` prop instead'),
  title: PropTypes.string,
  width: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
  height: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
}

BurgerIcon.defaultProps = {
  mainColor: COLORS.font1,
  hoverColor: COLORS.primary1,
  isActive: false,
  title: null,
  width: 12,
  height: 10,
  isAnimatedOnHover: false,
}
