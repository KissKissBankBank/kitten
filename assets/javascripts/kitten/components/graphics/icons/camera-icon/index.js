import React from 'react'
import PropTypes from 'prop-types'

export const CameraIcon = ({ color, title, ...props }) => (
  <svg
    width="20"
    height="16"
    viewBox="0 0 20 16"
    xmlns="http://www.w3.org/2000/svg"
    fill={color}
    {...props}
  >
    {title && <title>{title}</title>}
    <path
      d="M13 0l2 3h3a2 2 0 0 1 2 2v9a2 2 0 0 1-2 2H2a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h3l2-3h6zm-3 5.5a4 4 0 1 0 0 8 4 4 0 0 0 0-8zm0 2a2 2 0 1 1 0 4 2 2 0 0 1 0-4zM16.5 5a1.5 1.5 0 1 0 0 3 1.5 1.5 0 0 0 0-3zM12 1H8L7 3h6l-1-2z"
      fillRule="evenodd"
    />
  </svg>
)

CameraIcon.propTypes = {
  color: PropTypes.string,
  title: PropTypes.string,
}

CameraIcon.defaultProps = {
  color: '#222',
  title: '',
}
