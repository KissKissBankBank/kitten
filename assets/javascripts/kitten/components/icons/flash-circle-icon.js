import React from 'react'
import PropTypes from 'prop-types'
import COLORS from '../../constants/colors-config'
import { computeFromRatio } from '../../helpers/utils/ratio'

const DEFAULT_WIDTH = 20
const DEFAULT_HEIGHT = 20

export const FlashCircleIcon = ({
  circleColor,
  flashColor,
  width,
  height,
  title,
  ...props
}) => {
  const computed = computeFromRatio({
    defaultWidth: DEFAULT_WIDTH,
    defaultHeight: DEFAULT_HEIGHT,
    width,
    height,
  })

  return (
    <svg
      xmlns="http://www.w3.org/2000/svg"
      viewBox="0 0 20 20"
      width={computed.width}
      height={computed.height}
      {...props}
    >
      {title && <title>{title}</title>}
      <circle fill={circleColor} cx="10" cy="10" r="10" />
      <path
        d="m6 10 6-6-1.25 6h3.25l-6 6 1.25-6z"
        fill={flashColor}
        fillRule="evenodd"
      />
    </svg>
  )
}

FlashCircleIcon.prototype = {
  circleColor: PropTypes.string,
  flashColor: PropTypes.string,
  title: PropTypes.string,
  width: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
  height: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
}

FlashCircleIcon.defaultProps = {
  circleColor: COLORS.background1,
  flashColor: COLORS.font1,
}
