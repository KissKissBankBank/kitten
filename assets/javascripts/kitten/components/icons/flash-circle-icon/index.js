import React from 'react'
import PropTypes from 'prop-types'
import COLORS from '../../../constants/colors-config'
import { computeFromRatio } from '../../../helpers/utils/ratio'
import deprecated from 'prop-types-extra/lib/deprecated'
import classNames from 'classnames'


const DEFAULT_WIDTH = 20
const DEFAULT_HEIGHT = 20

export const FlashCircleIcon = ({
  bgColor,
  color,
  circleColor,
  flashColor,
  width,
  height,
  title,
  className,
  ...props
}) => {
  const computed = computeFromRatio({
    defaultWidth: DEFAULT_WIDTH,
    defaultHeight: DEFAULT_HEIGHT,
    width,
    height,
  })

  return (
    <svg
      xmlns="http://www.w3.org/2000/svg"
      viewBox="0 0 20 20"
      width={computed.width}
      height={computed.height}
      className={classNames('k-ColorSvg', className)}
      {...props}
    >
      {title && <title>{title}</title>}
      <circle fill={circleColor || bgColor} cx="10" cy="10" r="10" />
      <path
        d="m6 10 6-6-1.25 6h3.25l-6 6 1.25-6z"
        fill={flashColor || color}
        fillRule="evenodd"
      />
    </svg>
  )
}

FlashCircleIcon.prototype = {
  bgColor: PropTypes.string,
  color: PropTypes.string,
  circleColor: deprecated(
    PropTypes.string,
    '`circleColor` is deprecated. Please use `bgColor` instead.',
  ),
  flashColor: deprecated(
    PropTypes.string,
    '`flashColor` is deprecated. Please use `color` instead.',
  ),
  title: PropTypes.string,
  width: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
  height: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
}

FlashCircleIcon.defaultProps = {
  bgColor: COLORS.background1,
  color: COLORS.font1,
}
