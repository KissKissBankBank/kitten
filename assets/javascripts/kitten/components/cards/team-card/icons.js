import React, { Component, Fragment } from 'react'
import Radium from 'radium'
import PropTypes from 'prop-types'
import { ButtonIcon } from 'kitten/components/buttons/button-icon'
import { LinkedinIcon } from 'kitten/components/icons/linkedin-icon'
import { TwitterIcon } from 'kitten/components/icons/twitter-icon'
import { Marger as MargerBase } from 'kitten/components/layout/marger'
import { EmailIcon } from 'kitten/components/icons/email-icon'
import { TeamCardPhoneIcon } from 'kitten/components/cards/team-card/phone-icon'
import deprecated from 'prop-types-extra/lib/deprecated'

const Marger = Radium(MargerBase)

export class TeamCardIcons extends Component {
  static propTypes = {
    email: PropTypes.string,
    phoneNumber: PropTypes.string,
    links: PropTypes.oneOf(['linkedin', 'twitter']),

    // Deprecated.
    socialLink: deprecated(PropTypes.string, 'Prefer use links prop'),
  }

  static defaultProps = {
    email: '',
    phoneNumber: '',
    links: [],
  }

  getSocialLinks = () => {
    const { links, socialLink } = this.props

    if (links) return links

    // handle deprecated `socialLink` prop
    if (socialLink) return [{ name: 'linkedin', href: socialLink }]
  }

  render() {
    const { email, phoneNumber, links } = this.props

    return (
      <Marger top="1.5" style={styles.icons}>
        {email && (
          <ButtonIcon
            tag="a"
            href={`mailto:${email}`}
            modifier="hydrogen"
            className="k-ButtonIcon--phone"
            style={{ marginRight: 15 }}
          >
            <EmailIcon className="k-ButtonIcon__svg" />
          </ButtonIcon>
        )}

        {phoneNumber && <TeamCardPhoneIcon {...this.props} />}

        <SocialLinks links={this.getSocialLinks()} />
      </Marger>
    )
  }
}

class SocialLinks extends Component {
  render() {
    const { links } = this.props

    return (
      <Fragment>
        {links.map((link, index) => {
          let buttonStyle
          if (index !== links.length - 1) {
            buttonStyle = { marginRight: 15 }
          }

          return (
            <ButtonIcon
              tag="a"
              key={link.name}
              href={link.href}
              modifier={link.name}
              size="tiny"
              target="_blank"
              rel="noopener"
              style={buttonStyle}
            >
              {link.name === 'linkedin' && (
                <LinkedinIcon className="k-ButtonIcon__svg" />
              )}
              {link.name === 'twitter' && (
                <TwitterIcon className="k-ButtonIcon__svg" />
              )}
            </ButtonIcon>
          )
        })}
      </Fragment>
    )
  }
}

const styles = {
  icons: {
    display: 'flex',
  },
}
