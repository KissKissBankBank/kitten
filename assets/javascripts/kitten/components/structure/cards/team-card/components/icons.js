import React from 'react'
import styled from 'styled-components'
import PropTypes from 'prop-types'
import { pxToRem } from '../../../../../helpers/utils/typography'
import { Button } from '../../../../action/button'
import { LinkedinIcon } from '../../../../graphics/icons/linkedin-icon'
import { TwitterIcon } from '../../../../graphics/icons/twitter-icon'
import { EmailIcon } from '../../../../graphics/icons/email-icon'
import { TeamCardPhoneIcon } from './phone-icon'
import deprecated from 'prop-types-extra/lib/deprecated'

const StyledTeamCardIcons = styled.div`
  margin-top: ${pxToRem(15)};
  display: flex;

  > * {
    flex-shrink: 0;
  }

  gap: ${pxToRem(15)};
`

export const TeamCardIcons = ({
  email,
  phoneNumber,
  links,
  socialLink,
  ...props
}) => {
  const getSocialLinks = () => {
    if (links.length > 0) return links

    // handle deprecated `socialLink` prop
    if (socialLink) return [{ name: 'linkedin', href: socialLink }]
  }

  return (
    <StyledTeamCardIcons>
      {email && (
        <Button
          size="small"
          fit="icon"
          tag="a"
          href={`mailto:${email}`}
          modifier="hydrogen"
          aria-label="E-mail"
        >
          <EmailIcon aria-hidden />
        </Button>
      )}

      {phoneNumber && (
        <TeamCardPhoneIcon {...props} phoneNumber={phoneNumber} />
      )}

      <SocialLinks links={getSocialLinks()} />
    </StyledTeamCardIcons>
  )
}

const SocialLinks = ({ links = [] }) => (
  <>
    {links.map(link => {
      if (!link.href) return

      return (
        <Button
          tag="a"
          fit="icon"
          key={link.name}
          aria-label={link.name}
          href={link.href}
          modifier="hydrogen"
          target="_blank"
          rel="noopener"
          size="small"
        >
          {link.name === 'linkedin' && <LinkedinIcon aria-hidden />}
          {link.name === 'twitter' && <TwitterIcon aria-hidden />}
        </Button>
      )
    })}
  </>
)

TeamCardIcons.propTypes = {
  email: PropTypes.string,
  phoneNumber: PropTypes.string,
  links: PropTypes.arrayOf(
    PropTypes.shape({
      name: PropTypes.oneOf(['linkedin', 'twitter']),
      href: PropTypes.string,
    }),
  ),

  // Deprecated.
  socialLink: deprecated(PropTypes.string, 'Prefer use links prop'),
}

TeamCardIcons.defaultProps = {
  email: '',
  phoneNumber: '',
  links: [],
}
