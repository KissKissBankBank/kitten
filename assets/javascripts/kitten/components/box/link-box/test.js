import React from 'react'
import renderer from 'react-test-renderer'
import { LinkBox } from './index'
import { FlashCircleIcon } from '../../icons/flash-circle-icon'

describe('Deprecated <LinkBox />', () => {
  let component

  describe('by default', () => {
    beforeEach(() => {
      component = renderer.create(<LinkBox title="My title" />).toJSON()
    })

    it('matches with snapshot', () => {
      expect(component).toMatchSnapshot()
    })
  })

  describe('with some props', () => {
    beforeEach(() => {
      component = renderer
        .create(
          <LinkBox
            title="Lorem ipsum"
            text="In enim justo, rhoncus ut,"
            displayIcon
            href="#foobar"
            isExternal
          />,
        )
        .toJSON()
    })

    it('matches with snapshot', () => {
      expect(component).toMatchSnapshot()
    })
  })
})

describe('<LinkBox />', () => {
  let component

  describe('by default', () => {
    beforeEach(() => {
      component = renderer.create(<LinkBox />).toJSON()
    })

    it('matches with snapshot', () => {
      expect(component).toMatchSnapshot()
    })
  })

  describe('with some props', () => {
    beforeEach(() => {
      component = renderer
        .create(
          <LinkBox href="#foobar" isExternal>
            <LinkBox.Icon>
              <FlashCircleIcon />
            </LinkBox.Icon>

            <LinkBox.Text>foobar</LinkBox.Text>
          </LinkBox>,
        )
        .toJSON()
    })

    it('matches with snapshot', () => {
      expect(component).toMatchSnapshot()
    })
  })
})
