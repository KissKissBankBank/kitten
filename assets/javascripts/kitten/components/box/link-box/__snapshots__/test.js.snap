// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LinkBox /> by default matches with snapshot 1`] = `
<a
  className="sc-bdVaJa kKCOBn"
  href="#"
>
  <div
    className="k-LinkBox__link"
  >
    <div
      className="k-LinkBox__arrow"
    >
      <svg
        className=""
        fill="#222"
        height="10"
        style={
          Object {
            "transform": "rotate(90deg)",
          }
        }
        viewBox="0 0 8.48 5.64"
        width="10"
        xmlns="http://www.w3.org/2000/svg"
      >
        
        <path
          d="M0 4.24 L4.24,0 L8.48,4.24 L7.08,5.64 L4.24,2.77 L1.4,5.6 z"
        />
      </svg>
    </div>
  </div>
</a>
`;

exports[`<LinkBox /> with some props matches with snapshot 1`] = `
<a
  className="sc-bdVaJa kKCOBn"
  href="#foobar"
  rel="noopener"
  target="_blank"
>
  <div
    className="k-LinkBox__link"
  >
    <div
      className="k-LinkBox__icon"
    >
      <svg
        height={20}
        viewBox="0 0 20 20"
        width={20}
        xmlns="http://www.w3.org/2000/svg"
      >
        <circle
          cx="10"
          cy="10"
          fill="#fff"
          r="10"
        />
        <path
          d="m6 10 6-6-1.25 6h3.25l-6 6 1.25-6z"
          fill="#222"
          fillRule="evenodd"
        />
      </svg>
    </div>
    <div
      className="k-u-margin-top-double k-u-margin-bottom-double k-LinkBox__textContainer"
    >
      foobar
    </div>
    <div
      className="k-LinkBox__arrow"
    >
      <svg
        className=""
        fill="#222"
        height="10"
        style={
          Object {
            "transform": "rotate(90deg)",
          }
        }
        viewBox="0 0 8.48 5.64"
        width="10"
        xmlns="http://www.w3.org/2000/svg"
      >
        
        <path
          d="M0 4.24 L4.24,0 L8.48,4.24 L7.08,5.64 L4.24,2.77 L1.4,5.6 z"
        />
      </svg>
    </div>
  </div>
</a>
`;

exports[`Deprecated <LinkBox /> by default matches with snapshot 1`] = `
<a
  className="sc-bdVaJa kKCOBn"
  href="#"
  title="My title"
>
  <div
    className="k-LinkBox__link"
  >
    <div
      className="k-LinkBox__arrow"
    >
      <svg
        className=""
        fill="#222"
        height="10"
        style={
          Object {
            "transform": "rotate(90deg)",
          }
        }
        viewBox="0 0 8.48 5.64"
        width="10"
        xmlns="http://www.w3.org/2000/svg"
      >
        
        <path
          d="M0 4.24 L4.24,0 L8.48,4.24 L7.08,5.64 L4.24,2.77 L1.4,5.6 z"
        />
      </svg>
    </div>
  </div>
</a>
`;

exports[`Deprecated <LinkBox /> with some props matches with snapshot 1`] = `
<a
  className="sc-bdVaJa kKCOBn"
  href="#foobar"
  rel="noopener"
  target="_blank"
>
  <div
    className="k-LinkBox__link k-LinkBox__link--deprecated"
  >
    <div
      className="k-LinkBox__icon k-LinkBox__icon--deprecated k-u-hidden@xs-down--important"
    />
    <div
      className="k-LinkBox__textContainer k-LinkBox__textContainer--deprecated k-u-margin-top-double k-u-margin-bottom-double"
    >
      <div
        className="sc-htpNat bkOirm"
      >
        <span
          className="sc-bwzfXH uTxCW k-u-color-font1 k-u-size-default k-u-weight-regular k-u-line-height-1"
        >
          Lorem ipsum
        </span>
      </div>
      <span
        className="sc-bwzfXH uTxCW k-u-color-font1 k-u-size-tiny k-u-weight-light k-u-line-height-1-3"
      >
        In enim justo, rhoncus ut,
      </span>
    </div>
    <div
      className="k-LinkBox__arrow"
    >
      <svg
        className="k-ArrowIcon k-ArrowIcon--right k-ButtonIcon__svg"
        height="5.64"
        viewBox="0 0 6 6"
        width="8.48"
        xmlns="http://www.w3.org/2000/svg"
      >
        
        <path
          d="M6 0H0v6h2V2h4z"
        />
      </svg>
    </div>
  </div>
</a>
`;
